app/gui2/src/components/GraphEditor.vue

```diff
@@ -125,7 +128,7 @@ interface GraphStoredState {
   /** Whether or not the documentation panel is open. */
   doc: boolean
   /** The selected tab in the right-side panel. */
-  rtab: string
+  rtab: 'docs' | 'help'
```

Comment:
This data is persisted, and may have been serialized with a different version of the application. To avoid the type declaration providing guarantees that are not accurate, we should either declare a more general type or enforce the restricted type before using deserialized data.